graph/path/shortest.go

```diff
@@ -264,19 +282,30 @@ func (p *ShortestAlts) add(u graph.Node) int {
 	return idx
 }
 
+// set sets the weight of the path from the node in p.nodes indexed by mid to the node
+// indexed by to.
 func (p ShortestAlts) set(to int, weight float64, mid int) {
 	p.dist[to] = weight
 	p.next[to] = []int{mid}
 	if weight < 0 {
-		c, ok := p.cycCosts[[2]int{mid, to}]
+		c, ok := p.negCosts[[2]int{mid, to}]
 		if !ok {
-			p.cycCosts[[2]int{mid, to}] = weight
+			p.negCosts[[2]int{mid, to}] = weight
 		} else if weight < c {
-			p.cycCosts[[2]int{mid, to}] = math.Inf(-1)
+			// The only ways that we can have a new weight that is
```

Comment:
"two ways"

Reply:
"only" is used here as an emphatic. I think this is the correct wording.